tests/common/utils/test_docs_generator.py

```diff
@@ -66,5 +68,30 @@ def test_sorting_by_resource_id(self):
         self.assertEqual(sorted_list[2], ['CKV_AWS_1', '', 'aws_ebs_volume', '', ''])
         self.assertEqual(sorted_list[3], ['CKV_AWS_1', '', 'aws_s3_bucket', '', ''])
 
+
+def test_get_check_link():
+    # given
+    abs_path = "/path/to/checkov/checkov/terraform/checks/resource/aws/LambdaXrayEnabled.py"
+
+    # when
+    link = get_check_link(absolute_path=abs_path)
+
+    # then
+    assert link == "https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/aws/LambdaXrayEnabled.py"
+
+
+def test_get_check_link_for_markdown(mocker: MockerFixture):
+    # given
+    abs_path = "/path/to/checkov/checkov/terraform/checks/resource/aws/LambdaXrayEnabled.py"
+
+    mocker.patch("checkov.common.util.docs_generator.CREATE_MARKDOWN_HYPERLINKS", return_value=True)
+
+    # when
+    link = get_check_link(absolute_path=abs_path)
+
+    # then
+    assert link == "[LambdaXrayEnabled.py](https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/aws/LambdaXrayEnabled.py)"
```

Comment:
I think it can be a relative link, no?

Reply:
it would be just inside GitHub, then you would be right, but we also publish it under checkov.io and then it wouldn't work. So, I decided to do it this way and it will always work 😄